digikeyscraper/core.py

```diff
@@ -20,13 +20,19 @@
     + "(KHTML, like Gecko) Chrome/71.0.3578.98 Safari/537.36"
 )
 
+# Define timeout value (in seconds) for use by all GET Requests
+timeout = 10
```

Comment:
Make constants ALL_CAPS (see [pep8](https://www.python.org/dev/peps/pep-0008/#constants)).
```suggestion
TIMEOUT = 10
```

(you'll need to update the code where this is constant is used, too)